models/intermediate/int_shopify__customer_email_rollup.sql

```diff
@@ -22,20 +22,19 @@ with customers as (
         min(created_timestamp) as first_account_created_at,
         max(created_timestamp) as last_account_created_at,
         max(updated_timestamp) as last_updated_at,
-        max(accepts_marketing_updated_at) as accepts_marketing_last_updated_at,
+        max(marketing_consent_updated_at) as marketing_consent_updated_at,
         max(_fivetran_synced) as last_fivetran_synced,
         sum(orders_count) as orders_count,
         sum(total_spent) as total_spent,
 
         -- take true if ever given for boolean fields
-        {{ fivetran_utils.max_bool("has_accepted_marketing") }} as has_accepted_marketing,
```

Comment:
ah yes so i consolidated it with the new field in [staging](https://github.com/fivetran/dbt_shopify_source/pull/47/files#diff-2c50e9f75d5261e5a423647cd1aed38984e7fad2e70a1f90c115401dca3e6042R45-R51) -- i removed it from this section because it is no longer a boolean field, but a string. so it'll be included at the end of this CTE

Reply:
Perfect, thanks!